morl_baselines/common/performance_indicators.py

```diff
@@ -47,3 +47,26 @@ def sparsity(front: List[np.ndarray]) -> float:
     sparsity_value /= len(front) - 1
 
     return sparsity_value
+
+
+def eum(front: List[np.ndarray], weights_set: np.ndarray, utility: Callable = np.dot) -> float:
```

Comment:
I would name the function `expected_utility`, as many people don't know the EUM abbreviation.